src/components/layers-panel/layers-panel.tsx

```diff
@@ -0,0 +1,256 @@
+import { useState } from "react";
+import styled, { css } from "styled-components";
+import { lineGrey, senaryGrey } from "../../constants/colors";
+import { ListItemType } from "../../utils/enums";
+import { getCurrentLayoutProperty, useAppLayout } from "../../utils/layout";
+import { ListItem } from "../list-item/list-item";
+import { PlusButton } from "../plus-button/plus-button";
+
+enum Tabs {
+  Layers,
+  MapOptions,
+}
+
+type LayersPanelProps = {
+  id: string;
+  layers: any[];
+  type: ListItemType;
+  baseMaps: any[];
+  multipleSelection: boolean;
+  onLayerInsert: () => void;
+  onSceneInsert: () => void;
+  onBaseMapInsert: () => void;
+  onClose: () => void;
+};
+
+type TabProps = {
+  active: boolean;
+};
+
+type LayoutProps = {
+  layout: string;
+};
+
+const Container = styled.div<LayoutProps>`
+  display: flex;
+  flex-direction: column;
+  width: 359px;
+  background: ${(props) => props.theme.colors.panelBackgroundColor};
+  border-radius: 8px;
+  padding-bottom: 26px;
+
+  max-height: ${getCurrentLayoutProperty({
+    default: "408px",
+    tablet: "408px",
+    mobile: "calc(50vh - 82px)",
+  })};
+`;
+
+const PanelHeader = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: transparent;
+  position: relative;
+  border-radius: 8px;
+  margin-top: 20px;
+  gap: 32px;
+`;
+
+const Tab = styled.div<TabProps>`
+  position: relative;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 19px;
+  cursor: pointer;
+  color: ${({ theme }) => theme.colors.fontColor};
+
+  &:hover {
+    color: ${senaryGrey};
+    &::after {
+      background: ${senaryGrey};
+    }
+  }
+
+  ${({ active, theme }) =>
+    active &&
+    css`
+      &::after {
```

Comment:
It is better to use pseudo element here because of:
1. We can precisely manipulate the top underline position.
3. We don't need to change original Tab height to manipulate the top underline position.
4. We don't need to change original Tab width to manipulate the width of underline position.